Source/WebInspectorUI/UserInterface/Views/LogContentView.js

```diff
@@ -89,6 +89,14 @@ WI.LogContentView = class LogContentView extends WI.ContentView
             this._emulateUserGestureNavigationItemGroup = new WI.GroupNavigationItem([this._emulateUserGestureNavigationItem, new WI.DividerNavigationItem]);
         }
 
+        // 'Show Timestamp' toggle
+        this._showTimestampsNavigationItem = new WI.CheckboxNavigationItem("show-timestamps", WI.UIString("Show Timestamps", "Show Timestamps @ Console", "Checkbox shown in the Console to show timestamps."), WI.settings.showTimestamps.value);
```

Comment:
part of me feels like this should be in the Settings Tab since it's probably not super likely that a developer will want to toggle this on/off frequently

@patrickangle wdyt?

Reply:
Unless we (@dcrousso, @mokjasmine) feel there is a compelling use case for being able to quick toggle these on/off, that would be fine with me.

Reply:
yeah thinking on this more I'd probably lean towards moving this to the Console pane of the Settings Tab.

maybe we'd just have a "Show:" with a "[ ] Timestamps" item next to it (kinda like what we do for whitespace)?